packages/clarity/package.json

```diff
@@ -48,7 +49,8 @@
     "source-map-support": "^0.5.12",
     "ts-node": "^8.2.0",
     "tslint": "^5.17.0",
-    "typescript": "^3.5.1"
+    "typedoc": "^0.19.2",
+    "typescript": "^4.0.5"
```

Comment:
This is out of sync with the package-lock.json files and breaking CI. Once that is fixed, this looks good to merge

Reply:
I have commited updated package-lock.json files